os/include/elf32.h

```diff
@@ -74,41 +74,41 @@
 
 /* Values for Elf32_Ehdr::e_type */
 
-#define ET_NONE           0       /* No file type */
```

Comment:
Before is not a violation. After can be. Because there are spaces before tab. Let's revert this change.

Reply:
This change was done by cformarter.sh .. 
Is it wrong?

Reply:
As far as I know, if there is a space before a tab. CRC recognizes it a violation. But I can be incorrect. And I actually prefer spaces than tabs for alignment.